<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewerrlist2</class>
 <widget class="QDialog" name="viewerrlist2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>故障记录</string>
  </property>
  <widget class="QPushButton" name="buttontemp3">
   <property name="geometry">
    <rect>
     <x>534</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelrevisetype">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>90</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BCU监控记录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp0">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>345</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>90</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车辆</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonquit">
   <property name="geometry">
    <rect>
     <x>862</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>退   出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp1">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp4">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>10</y>
     <width>314</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>故障记录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelbuttonarea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>703</y>
     <width>1004</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttontemp2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>708</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_3">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>10</y>
     <width>345</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>16年06月17日 10:16:20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelcarId">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>90</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4408</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_1_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>180</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_1_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>180</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>故障种类</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_2_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>215</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_2_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>215</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_2_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>215</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_3_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>250</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_3_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>250</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_3_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>250</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_4_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>285</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_4_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>285</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_4_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>285</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_5_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>320</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_5_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>320</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_5_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>320</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_6_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>355</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_6_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>355</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_6_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>355</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_7_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>390</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_7_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>390</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_7_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>390</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_8_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>425</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_8_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>425</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_8_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>425</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_9_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>460</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_9_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>460</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_9_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>460</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_10_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>495</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_10_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>495</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_10_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>495</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_11_1">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>530</y>
     <width>50</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_11_3">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>530</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo_11_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>530</y>
     <width>260</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonnextpage">
   <property name="geometry">
    <rect>
     <x>862</x>
     <y>653</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>下   页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonpreviouspage">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>653</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>前   页</string>
   </property>
  </widget>
  <zorder>labelbuttonarea</zorder>
  <zorder>buttontemp3</zorder>
  <zorder>labelrevisetype</zorder>
  <zorder>buttontemp0</zorder>
  <zorder>title</zorder>
  <zorder>label</zorder>
  <zorder>buttonquit</zorder>
  <zorder>buttontemp1</zorder>
  <zorder>buttontemp4</zorder>
  <zorder>title_2</zorder>
  <zorder>buttontemp2</zorder>
  <zorder>title_3</zorder>
  <zorder>labelcarId</zorder>
  <zorder>labelinfo_1_2</zorder>
  <zorder>labelinfo_1_3</zorder>
  <zorder>labelinfo_2_1</zorder>
  <zorder>labelinfo_2_2</zorder>
  <zorder>labelinfo_2_3</zorder>
  <zorder>labelinfo_3_3</zorder>
  <zorder>labelinfo_3_2</zorder>
  <zorder>labelinfo_3_1</zorder>
  <zorder>labelinfo_4_1</zorder>
  <zorder>labelinfo_4_2</zorder>
  <zorder>labelinfo_4_3</zorder>
  <zorder>labelinfo_5_3</zorder>
  <zorder>labelinfo_5_2</zorder>
  <zorder>labelinfo_5_1</zorder>
  <zorder>labelinfo_6_3</zorder>
  <zorder>labelinfo_6_1</zorder>
  <zorder>labelinfo_6_2</zorder>
  <zorder>labelinfo_7_1</zorder>
  <zorder>labelinfo_7_2</zorder>
  <zorder>labelinfo_7_3</zorder>
  <zorder>labelinfo_8_1</zorder>
  <zorder>labelinfo_8_2</zorder>
  <zorder>labelinfo_8_3</zorder>
  <zorder>labelinfo_9_3</zorder>
  <zorder>labelinfo_9_1</zorder>
  <zorder>labelinfo_9_2</zorder>
  <zorder>labelinfo_10_1</zorder>
  <zorder>labelinfo_10_2</zorder>
  <zorder>labelinfo_10_3</zorder>
  <zorder>labelinfo_11_1</zorder>
  <zorder>labelinfo_11_3</zorder>
  <zorder>labelinfo_11_2</zorder>
  <zorder>buttonnextpage</zorder>
  <zorder>buttonpreviouspage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
